doc/relevancy.md

```diff
@@ -0,0 +1,460 @@
+| `searchable_text`        | ✖                  | New weighting query uses this for all_searchable_text and FIELD.no_stop | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | None                | None                         | ✔                | ✔               | ✖             | ✖                 |
+| `with_index_synonyms`    | ✖                  | New weighting query uses this for FIELD.synonym                         | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | Index synonym list  | Synonyms                     | ✔                | ✔               | ✖             | ✖                 |
+| `with_search_synonyms`   | ✖                  | New weighting query uses this at search time                            | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | Search synonym list | Synonyms                     | ✔                | ✔               | ✖             | ✖                 |
+| `exact_match`            | ✖                  | Used by best_bet_exact_match_text                                       | ✔                | ✖            | standard                 | ✔    | ✔         | ✖            | ✖                   | ✖                 | ✖                   | ✖                            | ✖                | ✖               | ✖             | ✖                 |
+| `best_bet_stemmed_match` | ✖                  | Used for triggering best bets                                           | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | ✖                   | ✖                            | ✔                | ✔               | ✖             | ✖                 |
+| `spelling_analyzer`      | ✖                  | Used in spelling_text field                                             | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | ✖                   | ✖                            | ✖                | ✖               | shingle       | ✖                 |
+| `string_for_sorting`     | ✖                  | Used for storing a sortable subfield                                    | ✔                | ✔            | keyword                  | ✔    | ✔         | ✖            | ✖                   | ✖                 | ✖                   | ✖                            | ✖                | ✖               | ✖             | ✖                 |
+
+
```

Comment:
Why are these other ones defined but not used?

Reply:
Sure thing I will add a note